locale/en/about/community.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Community Committee
+layout: about.hbs
+---
+
+# Community Committee
+
+The Community Committee (CommComm) is a top-level committee in the Node.js Foundation focused on community-facing efforts. Code commits !== the only way to contribute!
```

Comment:
Do we want to abbreviate to `CommComm` or `CC`?

Reply:
I'd stick with CommComm, that's what it's called most of the time, from what I've seen

Reply:
I agree with sticking to CommComm.